lockbox-ios/Common/Resources/Constants.swift

```diff
@@ -8,5 +8,5 @@ struct Constant {
     static let clientID = "1b024772203a0849"
     static let oauthHost = "oauth.accounts.firefox.com"
     static let profileHost = "profile.accounts.firefox.com"
-    static let redirectURI = "lockbox://redirect.ios"
```

Comment:
We should file a bugzilla for this.  Will bring it up with the team to be sure of the right procedures.